### DISCUSSION TITLE
Scene-viewer doesn't render the model with an hdr environment-image in AR

### DISCUSSION THREAD

**Discussion by AndresSalomon1990** [google/model-viewer#2436] [Q&A] [answered]
Hello devs! I'm seeing some big differences between render a model with webxr and scene-viewer in AR. Precisely, I'm using hdr images (with the environment-attribute) to make it more real but it seems that scene-viewer render the model without it in AR. I paste here 3 snapshots of what I'm viewing so you can see the differences:

1- [Preview 360° image](https://drive.google.com/file/d/1jUFPgWEobjze9dFtJtIj1zgPQDdYEVT9/view?usp=sharing)
2- [AR with webxr](https://drive.google.com/file/d/1FZ44aQMknMW-ezwzSSFUsHg4dz0J8PhF/view?usp=sharing)
3- [AR with scene-viewer](https://drive.google.com/file/d/1itrBL5iuCB2ZjDo9Ej_ofyZh6JP8FWNE/view?usp=sharing)

My code is pretty simple:
`<model-viewer 
        id="model"
        src="./assets/Candelabro.glb"
        alt="3D Model" 
        auto-rotate 
        auto-rotate-delay="1000"
        rotation-per-second="20deg"
        camera-controls
        loading="eager"
        reveal="auto"
        ar
        ar-modes="scene-viewer webxr quick-look"
        ar-scale="auto"
        touch-action
        bounds="tight"
        shadow-intensity="1"
        exposure="1"
        environment-image="./assets/ulmer_muenster_1k.hdr"
        max-field-of-view="60deg"
        min-field-of-view="15deg">`

I'm using a Nokia 7.1 with Android 10 to test it. If you need any other detail, please let me know.

Hope you can help me to understand this! Thanks in advance.

**Comment by elalish** [comment, marked as answer]
That's correct: WebXR uses your supplied environment image while Scene Viewer uses ARCore's ambient lighting calculation of your space (like Quick Look does). We will add support for WebXR lighting soon, but it will be optional.

**Reply by AndresSalomon1990** [reply]
Thanks @elalish ! Good to know 😊
